src/frequenz/client/dispatch/_client.py

```diff
@@ -130,9 +130,16 @@ async def create(
         dry_run: bool = False,
         payload: dict[str, Any] | None = None,
         recurrence: RecurrenceRule | None = None,
-    ) -> None:
+        fetch_resulting_dispatch: bool = True,
+    ) -> None | Dispatch:
```

Comment:
My only thought was that it takes another round-trip, that's why I made it optional opt-out for now. I can make it always fetch it, sure.

Reply:
I guess we won't be using the create much for now, so it might be good to get the interface as it should be in the future now if the impact is almost zero.